.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+node_modules
```

Comment:
##Optional 

- [ ] Please, You need to add only one `node_modules` folder inside the .gitignore. not 2

Reply:
@OLIPLICHE I got it.